Firestore/Example/Tests/Integration/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(XCTest REQUIRED)
+enable_testing()
+
+xctest_add_bundle(firebase_firestore_objc_integration_tests
```

Comment:
The contents of this file should likely be wrapped in an `if(APPLE) ... endif()` to prevent this from breaking the Linux and Windows builds.